components/list/list.js

```diff
@@ -77,7 +75,7 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 
 	connectedCallback() {
 		super.connectedCallback();
-		this.addEventListener('d2l-list-items-showing-count-change', this._handleListItemsShowingCountChange);
+		this.addEventListener('d2l-list-item-showing-count-change', this._handleListItemShowingCountChange);
```

Comment:
There are no external consumers of this event or method, so renaming should be safe 😄.